app/scripts/modules/netflix/instance/titus/netflixTitusInstanceDetails.controller.js

```diff
@@ -47,7 +48,7 @@ module.exports = angular.module('spinnaker.netflix.instance.titus.controller', [
     this.getBastionAddressForAccount = (account, region) => {
       return accountService.getAccountDetails(account).then((details) => {
         this.bastionHost = details.bastionHost || 'unknown';
-        this.apiEndpoint = _.where(details.regions, {name: region})[0].endpoint;
+        this.apiEndpoint = _.filter(details.regions, {name: region})[0].endpoint;
```

Comment:
Agreed - as long as everything is _functional_ in this PR, we can go ahead and merge it.